OptionSelector/CategoriesTableViewController.m

```diff
@@ -0,0 +1,175 @@
+    languageCategory.name = @"Language";
+    languageCategory.options = @[
+                                 @"Chinese",
+                                 @"Spanish",
+                                 @"English",
+                                 @"Hindi",
+                                 @"Arabic",
+                                 @"Portugese",
+                                 @"Bengali",
+                                 @"Russian",
+                                 @"Japanese",
+                                 @"Lahnda",
+                                 @"Javanese",
+                                 @"German",
+                                 @"Korean",
+                                 @"French",
+                                 @"Telgu"
+                                 ];
+    
+    
+    CQCategory *highestMountainCategory = [[CQCategory alloc] init];
+    highestMountainCategory.name = @"Highest Mountain";
+    highestMountainCategory.options = @[
+                                       @"Mount Everest",
+                                       @"K2",
+                                       @"Kangchenjunga",
+                                       @"Lhotse",
+                                       @"Makalu",
+                                       @"Cho Oyu",
+                                       @"Dhaulagiri I",
+                                       @"Manaslu",
+                                       @"Nanga Parbat",
+                                       @"Annapurna I",
+                                       @"Gasherbrum I",
+                                       @"Broad Peak"
+                                       ];
+    
+    
+    CQCategory *innovativeCountryCategory = [[CQCategory alloc] init];
+    innovativeCountryCategory.name = @"Innovative Country";
+    innovativeCountryCategory.options = @[
+                                           @"Finland",
+                                           @"Switzerland",
+                                           @"Israel",
+                                           @"Japan",
+                                           @"United States",
+                                           @"Germany",
+                                           @"Sweden",
+                                           @"Netherlands",
+                                           @"Singapore",
+                                           @"Taiwan"
+                                           ];
+    
+ 
+     self.categories   = @[languageCategory, highestMountainCategory, innovativeCountryCategory];
+    
+}
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self initializeData];
+    
+    
+    self.navigationItem.title = @"Random Categories";
+    
+}
+
+- (void)viewDidAppear:(BOOL)animated {
+    [self.tableView reloadData];
+
+}
+
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
```

Comment:
You can leave this method out entirely if you're not using sections. The default is 1.